pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BrokerUserDefinedMessageHandlerFactory.java

```diff
@@ -98,15 +101,18 @@ public void onError(Exception e, ErrorCode errorCode, ErrorType errorType) {
     }
   }
 
-  private class QueryQuotaUpdateMessageHandler extends DefaultMessageHandler {
+  private class RefreshTableConfigMessageHandler extends MessageHandler {
+    final String _tableNameWithType;
 
-    public QueryQuotaUpdateMessageHandler(QueryQuotaUpdateMessage queryQuotaUpdateMessage,
-        NotificationContext context) {
-      super(queryQuotaUpdateMessage, context);
+    RefreshTableConfigMessageHandler(TableConfigRefreshMessage tableConfigRefreshMessage, NotificationContext context) {
+      super(tableConfigRefreshMessage, context);
+      _tableNameWithType = tableConfigRefreshMessage.getTableNameWithType();
     }
 
     @Override
     public HelixTaskResult handleMessage() {
+      // TODO: Fetch the table config here and pass it into the managers, or consider merging these 2 managers
```

Comment:
not sure why we should merge the 2 managers. Fetching table config, checking for disabled table, etc.  maybe yes.